### PR TITLE
46 book issue by librarian to student

### LINKED ISSUE #46: Book Issue by librarian to student
- model
  - id (String)
  - issuedBy (Liberian uuid (String))
  - Book uuid (String)
  - Book reffrence number (String)
  - Student uuid (String)
  - createdDate (LocalDateTime)
  - updatedDate (LocalDateTime)
  - updatedBy (LocalDateTime)
- enum [ISSUED, RETURN]
- dto
   - issuedBy (Liberian uuid (String))
  - Book uuid (String)
  - Book reffrence number (String)
  - Student uuid (String)

- repository

### PR DESCRIPTION
**Pull request by manas-chandan**
- [x] Enum add [ISSUED, RETURN]
- [x] Model add with required filed
- [x] Dto add with required filed
- [x] Repository add for book issue

### CODE REVIEW

File: backend/src/main/java/com/lms/backend/constants/BookIssueStatus.java
Comment (smruti-banty) [inline backend/src/main/java/com/lms/backend/constants/BookIssueStatus.java:1]: RETURNED

File: backend/src/main/java/com/lms/backend/dto/BookIssueRequestDto.java
Comment (smruti-banty) [inline backend/src/main/java/com/lms/backend/dto/BookIssueRequestDto.java:1]: liberianUuid to adminId
studentUuid to studentId

File: backend/src/main/java/com/lms/backend/model/BookIssue.java
Comment (smruti-banty) [inline backend/src/main/java/com/lms/backend/model/BookIssue.java:1]: change studentUuid, liberianUuid

File: backend/src/main/java/com/lms/backend/repository/BookIssueRepository.java
Comment (smruti-banty) [inline backend/src/main/java/com/lms/backend/repository/BookIssueRepository.java:1]: mention generics and primary key type

**Review by smruti-banty — CHANGES_REQUESTED** [review]
Change accordingly

File: backend/src/main/java/com/lms/backend/constants/BookIssueStatus.java
Comment (manas-chandan) [reply backend/src/main/java/com/lms/backend/constants/BookIssueStatus.java:1]: OK

**Comment by manas-chandan** [comment]
all changes requested done

File: backend/src/main/java/com/lms/backend/dto/BookIssueRequestDto.java
Comment (smruti-banty) [inline backend/src/main/java/com/lms/backend/dto/BookIssueRequestDto.java:1]: bookRefferenceNumber typo reference

File: backend/src/main/java/com/lms/backend/model/BookIssue.java
Comment (smruti-banty) [inline backend/src/main/java/com/lms/backend/model/BookIssue.java:1]: bookRefferenceNumber typo reference

**Review by smruti-banty — CHANGES_REQUESTED** [review]
change accordingly

**Comment by manas-chandan** [comment]
typo change from reffrence  to reference

File: backend/src/main/java/com/lms/backend/model/BookIssue.java
Comment (manas-chandan) [reply backend/src/main/java/com/lms/backend/model/BookIssue.java:1]: done

**Review by smruti-banty — APPROVED** [review]
LGTM.

**Pull request merged**